internal/information/build_info.go

```diff
@@ -0,0 +1,72 @@
+package information
```

Comment:
I'd prefer a `buildinformation` package name. The reason is that `information` is too generic. Information about what? This might open possibility to add all kinds of "information" in the package but I think we should be more specific so the package does not become an "information container"